### ISSUE TITLE
Configure entity start-offset & end-offset from character-level to word-level

### ISSUE THREAD

**Issue by ccoppa** [nlplab/brat#1318]
Hi, 
Is there a way to configure the annotation.conf or other avenue to change the start-offset and end-offset from the current character-level to word-level?
Thanks

**Comment by tobiasoleary** [comment]
@ccoppa I don't believe so. I think the reasons they for not having this feature is that there are different ways of tokenizing documents depending. Whitespace tokenizer, Spacy Tokenizer, Tweet Tokenizer. Should $1000 be two tokens, or word ['$', '1000'] or a single token ['$1000']. You would have to write a utility to post-process the .txt and .ann file after doing the annotation to convert to a different format. If you do or did write a utility consider doing a merge request into the tools directory of brat, since there seems to be at least two other interested in it.
